platform.md

```diff
@@ -591,8 +603,11 @@ The launcher:
 - SHOULD replace the lifecycle with the process in memory without forking it.
 
 ##### Process Selection
-- **If** zero positional arguments are provided to the **launcher**, the lifecycle
-    - MUST select the process with `type` equal to `<process-type>` from `<layers>/config/metadata.toml`
+- **If** the final path element in `$0`, matches the `type` of a buildpack provided process
+    - `<proces-type>` SHALL be the final path element in `$0`
+    - the lifecycle:
+        - MUST select the process with `type` equal to `<process-type>` from `<layers>/config/metadata.toml`
+        - MUST append any user-provided `<args>` to process `<args>`
```

Comment:
Maybe it's only me but this sentence is a bit confusing.

Reply:
Yeah, I was struggling to word this well. Any suggestions?

Reply:
What confuses me here is that we're using `<args>` twice. As far as I understand, `<args>` are the arguments that the user provides to the command and the lifecycle should use them as the arguments to the process.
If I'm right, maybe the following rephrasing will be clearer:
```
MUST append the user-provided `<args>` to the process arguments.
```

By the way, a dot is missing at the end of each line but it's also true in sentences that you didn't touch :)

Reply:
That makes a lot of sense! Let me know if it is clearer now.